### ISSUE TITLE
Error 500 w/ PHP 7.0

### ISSUE THREAD

**Issue by jarbro** [splitbrain/dokuwiki-plugin-swiftmail#31]
Would get the following error.

[10-Nov-2016 17:45:21 America/New_York] PHP Parse error:  syntax error, unexpected 'new' (T_NEW) in /var/www/dokuwiki/public_html/lib/plugins/swiftmail/action.php on line 44

Upon deleting the plugin error 500 goes away.

**Comment by s3zwirni** [comment]
The reason is the "&" on this an some other lines in action.php:

$smtp =& new Swift_Connection_SMTP(
                        $this->getConf('smtp_host'),
                        $this->getConf('smtp_port'),
                        $this->getConf('smtp_ssl')
                     );

**Comment by wwwald** [comment]
So, a temporary fix would be to remove the "&"? 
Apologies, I'm not very up-to-date with respect to PHP7.

**Comment by s3zwirni** [comment]
Yes. Take a look: http://php.net/manual/de/migration70.incompatible.php

**Comment by YesThatAllen** [comment]
Moving to https://github.com/splitbrain/dokuwiki-plugin-smtp seems to be preferred. Any pointers on how to view/migrate the old swiftmail configuration would be appreciated.

**Comment by RubyRachel** [comment]
[10-Nov-2016 17:45:21 America/New_York] PHP Parse error: syntax error, unexpected 'new' (T_NEW) in /var/www/dokuwiki/public_html/lib/plugins/swiftmail/action.php on line 44 how to solve this proplem
